### DISCUSSION TITLE
Welcome to discuss v0.6 version | 欢迎讨论 v0.6 版本

### DISCUSSION THREAD

**Discussion by elliotxx** [KusionStack/karpor#624] [General]
🎯 v0.6 version positioning:

After a period of iteration, [v0.5](https://github.com/KusionStack/karpor/milestone/4) version has been officially released! We are now officially starting the version planning for [v0.6](https://github.com/KusionStack/karpor/milestone/5). If v0.5 solidifies the basic features, security, and user experience, then v0.6 will focus on **feature enhancements** and **AI Features**, striving for "usability" and "eye catching". 

We have preliminarily planned the following features:
- AI based k8s risk problem solution, similar to k8sGPT 
- support for searching k8s resources through natural language 
- logs, event aggregator, real-time viewing of key resource logs and events on the UI interface
- ...

You can learn about all planned issues and overall progress in [v0.6 Milestone](https://github.com/KusionStack/karpor/milestone/5). If you have better suggestions, please feel free to leave a message below!

---

🎯 v0.6 版本定位：

经过一段时间的迭代后，[v0.5](https://github.com/KusionStack/karpor/milestone/4) 版本已经正式 Release！我们现在正式开始 [v0.6](https://github.com/KusionStack/karpor/milestone/5) 的版本规划，如果说 v0.5 夯实了基本功能、安全性和用户体验，那么 v0.6 将着重在**功能增强**和 **AI Feature** 上，力求”好用“和”眼前一亮“。

我们初步规划了以下功能：
- 基于 AI 的 k8s 风险问题解决方案，类似 k8sGPT
- 支持通过自然语言搜索 k8s 资源
- 日志、事件聚合器，可在 UI 界面实时查看关键资源的日志和事件
- ……

你可以在 [v0.6 Milestone](https://github.com/KusionStack/karpor/milestone/5) 中了解所有已规划的 Issue 和整体进展。如果你有更好的建议，欢迎在下方留言！

**Comment by yangyuw** [comment]
希望提供 openai 等接口调用的代理环境变量配置，毕竟很多集群都是在纯内网环境

**Reply by elliotxx** [reply]
@yangyuw  收到，多谢反馈，我们之前为 Docker Registry 添加过 [代理变量配置](https://www.kusionstack.io/karpor/getting-started/installation#image-registry-proxy-for-china)，LLM 服务的代理变量配置我记了个 issue #736 ，放到了下个版本迭代中：#723

**Reply by elliotxx** [reply]
如果在用 Karpor 的话可以在 #343 中留个言，给我们一些动力！

**Reply by elliotxx** [reply]
> 希望提供 openai 等接口调用的代理环境变量配置，毕竟很多集群都是在纯内网环境

@yangyuw 你希望怎么配置 openai 的代理？能提供更多信息吗

**Reply by elliotxx** [reply]
@yangyuw Hi 我们已经支持在 Helm 安装的时候指定 **OpenAI 等接口调用的代理环境变量配置**，可以看下能不能满足你的需求：
![image](https://github.com/user-attachments/assets/bb7f682c-1c59-4dca-b85c-1581b5e74d4d)

- 文档链接：https://www.kusionstack.io/karpor/next/getting-started/installation#enable-ai-features
- 需求 Issue：https://github.com/KusionStack/karpor/issues/736

**Comment by elliotxx** [comment]
v0.6.0 has been officially released 🎉
Release Note 👉 https://github.com/KusionStack/karpor/releases/tag/v0.6.0
